spec/support/page_objects/teacher_interface/active_application.rb

```diff
@@ -0,0 +1,37 @@
+module PageObjects
+  module TeacherInterface
+    class TaskListItem < SitePrism::Section
+      element :link, "a"
+      element :status, ".govuk-tag"
+    end
+
+    class TaskListSection < SitePrism::Section
+      element :heading, "h2"
+      sections :task_list_items, TaskListItem, ".app-task-list__item"
+    end
+
+    class ActiveApplication < SitePrism::Page
```

Comment:
I wonder about the naming of this page. We don't really have a concept of an "active" application, since an application is active for both a teacher and an assessor. For [the assessor interface we've simply called it `Application`](https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/main/spec/support/page_objects/assessor_interface/application.rb) and I think we can do the same here.

The pages themselves can be referenced as `teacher_application_page` and `assessor_application_page`, and I think that's actually better than `active_application_page` as just the name on its own is ambiguous as to whether it's for teachers or for assessors.